### DISCUSSION TITLE
Loosing data between sessions

### DISCUSSION THREAD

**Discussion by horusconsulting** [ltsp/ltsp#220] [Q&A]
Hello. I used K12 Linux for a while and I passed to new LTSP recently. I love it, but have also some issues, the main of which is loss of data between sessions. For example, a user sets up his personal e-mail in one terminal. It works fine. When he logs out and seats at another terminal, mail is gone and he has to set it up again from scratch. What's wrong?
Thanks for your help.
Gianugo Altieri, small business in Switzerland

**Comment by alkisg** [comment]
Data loss between sessions could mean that for some reason /home/username isn't mounted from the server.
Are you using the new LTSP or the old LTSP5?
What is the output of this command?

```shell
ltsp info
```
